parser/src/test/scala/hmda/parser/fi/ts/TsCsvParserSpec.scala

```diff
@@ -8,9 +8,32 @@ class TsCsvParserSpec extends PropSpec with PropertyChecks with MustMatchers wit
   property("Transmittal Sheet must be parsed from CSV") {
     forAll(tsGen) { (ts) =>
       whenever(ts.id == 1) {
-        TsCsvParser(ts.toCSV) mustBe ts
+        TsCsvParser(ts.toCSV).right.get mustBe ts
```

Comment:
I prefer `TsCsvParser(ts.toCSV) mustBe Right(ts)` here.
If you accept that one, consider analogous changes on other lines, too.